eternalcore-core/src/main/java/com/eternalcode/core/feature/itemedit/ItemLoreArgument.java

```diff
@@ -17,7 +17,7 @@
 @LiteArgument(type = int.class, name = ItemLoreArgument.KEY)
 class ItemLoreArgument extends AbstractViewerArgument<Integer> {
 
-    private static final List<Integer> suggestions = List.of(0, 1, 2, 3, 4, 5);
+    private static final List<Integer> suggestions = List.of(1, 2, 3, 4, 5, 6); // Sugeruj numery od 1 do 6
```

Comment:
Code comments, we write in English I think.
```suggestion
    private static final List<Integer> suggestions = List.of(1, 2, 3, 4, 5, 6); // Sort numbers from 1 to 6
```